tools/appveyor.psm1

```diff
@@ -174,11 +181,14 @@ function Invoke-AppVeyorInstall
 {
     if(Test-DailyBuild){
         $buildName = "[Daily]"
-        if($env:APPVEYOR_PULL_REQUEST_TITLE)
+
+        # Add daily to title if it's not already there
+        # It can be there already for rerun requests
+        if($env:APPVEYOR_PULL_REQUEST_TITLE -and $env:APPVEYOR_PULL_REQUEST_TITLE  -notmatch '^\[Daily\].*$')
         {
             $buildName += $env:APPVEYOR_PULL_REQUEST_TITLE
         }
-        else
+        elseif($env:APPVEYOR_REPO_COMMIT_MESSAGE  -notmatch '^\[Daily\].*$'  )
```

Comment:
Only when it's at the front, preventing this code from running twice (In the case of a rerun through the AppVeyor UI)